apps/emqx/test/emqx_persistent_session_SUITE.erl

```diff
@@ -50,13 +50,17 @@ groups() ->
     SnabbkaffeTCs = [TC || TC <- TCs, is_snabbkaffe_tc(TC)],
     GCTests  = [TC || TC <- TCs, is_gc_tc(TC)],
     OtherTCs = (TCs -- SnabbkaffeTCs) -- GCTests,
-    [ {persistent_store_enabled, [ {group, no_kill_connection_process}
-                                 , {group,    kill_connection_process}
```

Comment:
Q: this group is not being tested anymore.  Is this to avoid the flakiness?
(If so, should we delete the [config](https://github.com/emqx/emqx/blob/46788ad31b737978f114652094ab664b90a98f1d/apps/emqx/test/emqx_persistent_session_SUITE.erl#L122-L123) for this group?)

Reply:
Good catch! This was not intentional, but a refactoring mistake. I will add it back.

Btw, the only flakiness I am aware of is in the groups *_snabbkaffe.